File: python/_etc/make_folder2.py
import os

# 몇개의 폴더를 생성?
chap_num = 12
sect_num = 20

dir_name = "0-00"

# 폴더 만들기
for i in range(1, chap_num+1):
    for j in range(1, sect_num+1):
        dir_name = f"{i}-{j:02}"  # 8-09
        os.mkdir(dir_name)

print("폴더 생성을 완료하였습니다.")

File: javascript/_etc/makeFolder2.js
const fs = require('fs');

const makeFolder = (dirname) => {
  if (!fs.existsSync(dirname)) {
    fs.mkdirSync(dirname);
  }
};

// 몇개의 폴더를 생성?
const chapNum = 12
const sectNum = 20

let dirname = './0-00';

for (let i = 1; i <= chapNum; i++) {
  for (let j = 1; j < sectNum; j++) {
    if (j < 10)
      dirname = `./${i}-0${j}`; // 8-09
    else
      dirname = `./${i}-${j}`;  // 8-12
    makeFolder(dirname);
  }
}

console.log('폴더 생성을 완료하였습니다.');
